Configurations/50-win-onecore.conf

```diff
@@ -61,4 +61,50 @@ my %targets = (
         ex_libs         => "onecore.lib",
         multilib        => "-arm64",
     },
+
+    # Universal Windows Platform (UWP) App Support
+    
+    # TODO
+    #
+    # The 'disable' attribute should have 'uplink' and 'asm' as well...  
```

Comment:
I thought we decided "asm" was not required after all (except for old builds)?

Reply:
This doesn't actually quite make sense as written because you say in the TODO that it should have uplink in the disable attribute...but it does have it below! It just doesn't work...

Reply:
> I thought we decided "asm" was not required after all (except for old builds)?

I will modify it. But I think we should also note "But if you want to build library with Windows 10 Version 1809 SDK or earlier, the 'disable' attribute should also have 'asm'." to help the OpenSSL contributors know. What do you think about that?